web/app.js

```diff
@@ -176,6 +176,7 @@ var PDFViewerApplication = {
   preferencePdfBugEnabled: false,
   preferenceShowPreviousViewOnLoad: true,
   preferenceDefaultZoomValue: '',
+  preferenceEnhanceTextSelection: true,
```

Comment:
Let's enable it by default since it's not CPU hungry anymore for standard usage.